### PR TITLE
feat(adjust): update wrapper for Adjust Cordova SDK v4.35.1

### PR DESCRIPTION
**Pull request by uerceg**
- Updated `index.ts` to expose current state of Adjust Cordova SDK v4.35.1 to TypeScript users.

### CODE REVIEW

**Comment by uerceg** [comment]
@danielsogl 👋 Hello there, hope everything's running well on your end. Are you still maintaining the repo and is there any hope to see this PR checked / merged? Looking forward to hearing back from you. Cheers.

File: src/@awesome-cordova-plugins/plugins/adjust/index.ts [hunk 1]
```diff
@@ -401,6 +439,28 @@ export class AdjustAdRevenue {
   }
 }
 
+export class AdjustAppStorePurchase {
+  private receipt: string;
+  private receipt: string;
```
Comment (jfougere) [inline src/@awesome-cordova-plugins/plugins/adjust/index.ts:444]: Duplicated receipt field

**Review by danielsogl — APPROVED** [review]

**Comment by danielsogl** [comment]
@uerceg can you resolve the merge issues?

**Comment by uerceg** [comment]
Hey @danielsogl 

Sorry for spotting this a bit late, but as far as I can see, you have merged another PR to address the issues from this one.

Thank you very much, really appreciate it.

Have a great holiday season ahead.

Cheers! 🍺🎄

**Pull request closed without merging**